blob-clients/blob-client-jersey2/src/main/java/com/bazaarvoice/emodb/blob/client/BlobStoreClient.java

```diff
@@ -17,11 +17,10 @@
 import com.bazaarvoice.emodb.client.EmoResource;
 import com.bazaarvoice.emodb.client.EmoResponse;
 import com.bazaarvoice.emodb.client.uri.EmoUriBuilder;
+import com.bazaarvoice.emodb.common.api.ServiceUnavailableException;
+import com.bazaarvoice.emodb.common.api.UnauthorizedException;
 import com.bazaarvoice.emodb.common.json.RisonHelper;
-import com.bazaarvoice.emodb.sor.api.Audit;
-import com.bazaarvoice.emodb.sor.api.TableExistsException;
-import com.bazaarvoice.emodb.sor.api.TableOptions;
-import com.bazaarvoice.emodb.sor.api.UnknownTableException;
+import com.bazaarvoice.emodb.sor.api.*;
```

Comment:
This is a repeating mistake. Check before you raise a pr. 
Wild card imports is not good. refer [this](https://docs.google.com/document/d/1S-Zz626I_5-GVrq5Bm22Zv8qCkyikWWyoxdOat-BMwQ/edit).

Reply:
This actually happened because of the intelliJ ID, but issue addressed and pushed